components/Avatar/Avatar.jsx

```diff
@@ -2,19 +2,33 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-
-
-const Avatar = props => (
-  <span className={`avatar color-teal avatar--${props.size}`}>
-    <span
-      className='avatar-user user-avatar'
-      style={{
-        backgroundImage: `url('${props.image}')`,
-      }}
-    />
-
-  </span>
-);
+import styles from './Avatar.scss';
+import cx from 'classnames';
+
+const setClass = (props) => {
+  return cx({
+    [styles.defaultAvatarXsmall]: props.size === 'xsmall',
+    [styles.defaultAvatarSmall]: props.size === 'small',
+    [styles.defaultAvatarMedium]: props.size === 'medium',
+    [styles.defaultAvatarLarge]: props.size === 'large',
+    [styles.defaultAvatarXlarge]: props.size === 'xlarge',
+    [styles.defaultAvatarXxlarge]: props.size === 'xxlarge',
+  })
+}
+
+
+const Avatar = props => {
+  return (
+    <span className={setClass(props)} style={{ zIndex: 1 }}>
+      <span
```

Comment:
I have some updates to avatar coming in right here.

Reply:
Good to note.  Would you mind rebasing off of master once this gets in?